rpcs3/Emu/Cell/Modules/cellGame.h

```diff
@@ -307,50 +309,10 @@ enum
 	CELL_HDDGAME_SIZEKB_NOTCALC        = -1,
 };
 
-struct CellHddGameSystemFileParam
-{
-	char title[CELL_HDDGAME_SYSP_TITLE_SIZE];
-	char titleLang[CELL_HDDGAME_SYSP_LANGUAGE_NUM][CELL_HDDGAME_SYSP_TITLE_SIZE];
-	char titleId[CELL_HDDGAME_SYSP_TITLEID_SIZE];
-	char reserved0[2];
-	char dataVersion[CELL_HDDGAME_SYSP_VERSION_SIZE];
-	char reserved1[2];
-	be_t<u32> attribute;
-	be_t<u32> parentalLevel;
-	be_t<u32> resolution;
-	be_t<u32> soundFormat;
-	char reserved2[256];
-};
-
-struct CellHddGameCBResult
-{
-	be_t<s32> result;
-	be_t<s32> errNeedSizeKB;
-	vm::bptr<char> invalidMsg;
-	vm::bptr<void> reserved;
-};
-
-struct CellHddGameStatGet
-{
-	be_t<s32> hddFreeSizeKB;
-	be_t<u32> isNewData;
-	char contentInfoPath[CELL_HDDGAME_PATH_MAX];
-	char hddGamePath[CELL_HDDGAME_PATH_MAX];
-	char reserved0[2];
-	be_t<s64> atime;
-	be_t<s64> mtime;
-	be_t<s64> ctime;
-	CellHddGameSystemFileParam getParam;
-	be_t<s32> sizeKB;
-	be_t<s32> sysSizeKB;
-	char reserved1[68];
-};
-
-struct CellHddGameStatSet
-{
-	vm::bptr<CellHddGameSystemFileParam> setParam;
-	vm::bptr<void> reserved;
-};
+using CellHddGameStatGet = CellGameDataStatGet;
+using CellHddGameStatSet = CellGameDataStatSet;
+using CellHddGameSystemFileParam = CellGameDataSystemFileParam;
```

Comment:
This has a different size than CellGameDataSystemFileParam, which cascades through the other structs.
Idk how this will behave for older games, since the old structs seem to be a 1.9 thing

Reply:
They are the same functions and use the same structures, you can check out the disassembly. There was a padding mistake in CellHddGameSystemFileParam::reserved2[256], probably a copypaste from CellGameDataSystemFileParam. I updated reserved2 size accordinally.

Reply:
you mean the typo was in the docs?
I don't ave the 1.9 fw to check sadly